setup.py

```diff
@@ -7,15 +7,15 @@
 # NOTE: When updating botocore make sure to update awscli/boto3 versions below
 install_requires = [
     # pegged to also match items in `extras_require`
-    'botocore>=1.31.16,<1.31.71',
+    'botocore>=1.32.4,<1.33.2',
```

Comment:
I missed this in the review, it's doubtful this aiobotocore is compatible across these versions given it required changes to match the latest botocore